speclang_modern.tex

```diff
@@ -3291,6 +3313,7 @@ \subsection{Volatile variables}\label{sec:volatile-variables}
 
 
 \section{Undefined values, dangling pointers}
+\marginpar{It seems silly to have a composite section title and then separate subsections for each of the pieces. Perhaps just make the two subsections sections themselves.}
```

Comment:
Indeed, initialization and dangling pointers are quite orthogonal. The first apply when a local gets in scope and the latter when it exits scope.

Reply:
Made them sections.